apps/examples/hello/hello_main.c

```diff
@@ -68,5 +69,11 @@ int hello_main(int argc, char *argv[])
 #endif
 {
 	printf("Hello, World!!\n");
+#ifdef CONFIG_PM
+	/* grace period for system to settle */
+	MSLEEP(5000);
```

Comment:
5000ms is 5secs. then why don't you use sleep instead of making MSLEEP?

Reply:
ok